package.json

```diff
@@ -6,28 +6,27 @@
     "node": ">=20.0.0"
   },
   "scripts": {
-    "prepare": "gulp bundle --color",
-    "build": "gulp build --color",
+    "build": "gulp bundle docs:build --color --series",
+    "prestart": "npm run build",
     "start": "gulp --color",
-    "test": "npm run lint && npm run jest:ci",
+    "pretest": "npm run build",
+    "test": "jest --color --coverage",
```

Comment:
Nice, I think removing the `lint` test from `npm test` will also make it clearer whether the tests have actually failed or if it was just a code style issue (which is reported on separately anyway) on GitHub?

I previously proposed doing this a different way in #1084

Reply:
It's possibly still better to have `npm test` run both the code style checks and all the other tests, as that’s what people might run locally out of habit, but then separate out the GitHub workflows via separate commands in the GitHub config?

But either way is probably fine.

I previously split up the tests to run in separate GitHub actions in #1056, mainly to speed things up and make it clearer what failed.

Reply:
Could do! Although if you split them out more then it requires @anandamaryon1 or someone with admin rights to update the settings for which actions are required to pass for merging, so maybe hold off for now?